unity/Assets/Scripts/IK_Robot_Arm_Controller.cs

```diff
@@ -91,21 +102,8 @@ void Start()
     //     StopMotionOnContact = b;
     // }
 
-    //debug for gizmo draw
-#if UNITY_EDITOR
```

Comment:
Did you mean to remove this?

Reply:
It's not removed. Just moved to the bottom of the file inside of the other
```
#if UNITY_EDITOR
```
block.

https://github.com/allenai/ai2thor/blob/manipulathor-ux/unity/Assets/Scripts/IK_Robot_Arm_Controller.cs#L617